include/pstore/mcrepo/compilation.hpp

```diff
@@ -107,7 +107,7 @@ namespace pstore {
             /// The extent of the fragment referenced by this compilation symbol.
             extent<fragment> fext;
             typed_address<indirect_string> name;
-            linkage_type linkage;
+            repo::linkage linkage;
```

Comment:
Could it be `linkage` instead of ` repo::linkage`?

Reply:
That won't compile with gcc because the member name clashes.